### PR TITLE
Add Monica to team page

### PR DESCRIPTION
**Pull request by mnlumi**
Add Monica to team page.

### CODE REVIEW

File: themes/default/data/team/team/monica-rodriguez.toml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+id = "monica-rodriguez"
+name = "Monica Rodriguez"
+title = "Senior Product Manager"
+company = "Pulumi"
+weight = 1
```
Comment (mnlumi) [inline themes/default/data/team/team/monica-rodriguez.toml:5]: Is weight needed? Not everyone has that in their file.

**Comment by susanev** [comment]
@mnlumi ur image is over a 1 mb, it does not need to be that big. id be happy to push a commit to reduce the size if you'd like; otherwise pls reduce the image size, thank you!!

**Comment by mnlumi** [comment]
> @mnlumi ur image is over a 1 mb, it does not need to be that big. id be happy to push a commit to reduce the size if you'd like; otherwise pls reduce the image size, thank you!!

@susanev I've reduced the size. Thanks for looking at this!

**Review by susanev — APPROVED** [review]
welcome to pulumi!! 🎉

**Pull request merged**